fedot/api/api_utils/assumptions/task_assumptions.py

```diff
@@ -64,6 +64,8 @@ def builders(self):
             .add_branch('polyfit', 'lagged')
             .grow_branches(None, 'ridge')
             .join_branches('ridge'),
+            'topo_ica_ridge': PipelineBuilder().add_sequence('lagged', 'fast_topological_features',
+                                                             'fast_ica', 'ridge'),
```

Comment:
неконсистентность в переносах строк
```suggestion
            'topo_ica_ridge':
                PipelineBuilder()
            .add_sequence('lagged', 'fast_topological_features', 'fast_ica', 'ridge'),
```

Reply:
Почему? Вроде бы все ок.

Reply:
ну просто некоторые записи в словаре с переносом `PipelineBuilder()` на новую строку и дальнейшим переносом шагов билдера, а некоторые нет

Reply:
[3972d10](https://github.com/aimclub/FEDOT/pull/1252/commits/3972d10106bbc4f3fdf65b9e35e3c7ee5ba98e0b)

Сразу пофиксил и для другой модели в том же месте.